.github/ISSUE_TEMPLATE.MD

```diff
@@ -0,0 +1,27 @@
+## Expected Behavior
+<!--- Tell us what should happen-->
+
+## Current Behavior
+<!--- Tell us what currently happens instead of the expected behavior -->
+
+## Possible Solution (we welcome any logical suggestion)
+<!--- You have a solution which may work?  Please share it, we would like to see what you have in mind -->
```

Comment:
"Do you have a potential solution to the issue? Please share it; we'd like to see what you have in mind."